test/unit/mdc-select/mdc-select.test.js

```diff
@@ -115,77 +82,153 @@ test('#get/setDisabled', () => {
-});
-
-test('#initialSyncWithDOM sets the selected index if a menu item contains an aria-selected attribute', () => {
-  const menu = new FakeMenu();
-  const fixture = getFixture();
-  // Insert menu item into fixture to pretend like the fake menu items are part of the component under test.
-  menu.items[1].setAttribute('aria-selected', 'true');
-  fixture.appendChild(menu.items[1]);
-
-  const component = new MDCSelect(fixture, /* foundation */ undefined, () => menu);
+  assert.equal(component.value, 'orange');
+});
+
+test('#set value sets the value on the <select>', () => {
+  const {component, nativeControl} = setupTest();
+  component.value = 'orange';
+  assert.equal(nativeControl.value, 'orange');
+});
+
+test('#set value sets non value to empty string on <select>', () => {
+  const {component, nativeControl} = setupTest();
+  component.value = 'grape';
+  assert.equal(nativeControl.value, '');
+});
+
+test('#initialSyncWithDOM sets the selected index if an option has the selected attr', () => {
+  const fixture = bel`
+    <div class="mdc-select">
+      <div class="mdc-select__label">Pick a Food Group</div>
+      <div class="mdc-select__bottom-line"></div>
+      <select class="mdc-select__native-control">
+        <option value="orange">
+          Orange
+        </option>
+        <option value="apple" selected>
+          Apple
+        </option>
+      </select>
+    </div>
+  `;
+  const component = new MDCSelect(fixture, /* foundation */ undefined);
   assert.equal(component.selectedIndex, 1);
 });
 
-test('#initialSyncWithDOM disables the menu if aria-disabled="true" is found on the element', () => {
-  const fixture = getFixture();
-  fixture.setAttribute('aria-disabled', 'true');
+test('#initialSyncWithDOM disables the select if the disabled attr is found on the <select>', () => {
+  const fixture = bel`
+    <div class="mdc-select">
+      <div class="mdc-select__bottom-line"></div>
+      <select class="mdc-select__native-control" disabled>
+        <option value="orange">
+          Orange
+        </option>
+        <option value="apple" selected>
+          Apple
+        </option>
+      </select>
+    </div>
+  `;
   const component = new MDCSelect(fixture);
-  assert.isOk(component.disabled);
+  assert.isTrue(component.disabled);
 });
 
 test('adapter#addClass adds a class to the root element', () => {
   const {component, fixture} = setupTest();
   component.getDefaultFoundation().adapter_.addClass('foo');
-  assert.isOk(fixture.classList.contains('foo'));
+  assert.isTrue(fixture.classList.contains('foo'));
 });
 
 test('adapter#removeClass removes a class from the root element', () => {
   const {component, fixture} = setupTest();
   fixture.classList.add('foo');
   component.getDefaultFoundation().adapter_.removeClass('foo');
-  assert.isNotOk(fixture.classList.contains('foo'));
+  assert.isFalse(fixture.classList.contains('foo'));
+});
+
+test('does not create label', () => {
```

Comment:
We would never expect a component or adapter to create elements; we should instead test that `adapter_.floatLabel` doesn't throw.

There are some examples of this e.g. for helperText in mdc-text-field.test.js.